### ISSUE TITLE
Definition File for HAMTIDE constituents 

### ISSUE THREAD

**Issue by martin-rdz** [tsutterley/pyTMD#179]
Hi everybody,

I want to use the constituents provided by [hamtide](https://www.cen.uni-hamburg.de/en/icdc/data/ocean/hamtide.html) within pyTMD.
Basically, the data is provided as separate netcdf file for each constituent and split up into amplitude and current information.
Can someone please provide me with some guidance on how to setup the Definition Files?

Thank a lot in advance!
-Martin

**Comment by tsutterley** [comment]
Hi @martin-rdz,
I was hoping to have a quick reply to get you going with the definition files. However, it looks like HAMTIDE has a slightly different convention in terms of netCDF variable names. So I added `HAMTIDE11` as a supported model within PR #180. This uses the `FES.io` functions, but adds the new variable names. Tested the height outputs using the forecast notebook, and everything seemed reasonable compared with other models. Hope this helps.
Cheers,
-ts
